packages/trpc/server/routers/viewer/bookings/prevResponse.handler.ts

```diff
@@ -0,0 +1,31 @@
+import type { PrismaClient } from "@calcom/prisma";
+
+import type { TrpcSessionUser } from "../../../trpc";
+import type { TPrevResponseInputSchema } from "./prevResponse.schema";
+
+type PrevResponseOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+    prisma: PrismaClient;
+  };
+  input: TPrevResponseInputSchema;
+};
+
+export const prevResponseHandler = async ({ ctx, input }: PrevResponseOptions) => {
+  const { eventTypeId } = input;
+  const { user, prisma } = ctx;
+
+  return await prisma.booking.findFirst({
```

Comment:
We need the latest one and not the first one

Reply:
Yeah forgot to add orderBy, will fix it